tutors3/patches/openedx-common-settings

```diff
@@ -5,7 +5,7 @@ GRADES_DOWNLOAD["STORAGE_KWARGS"]["location"] = GRADES_DOWNLOAD["STORAGE_KWARGS"
 GRADES_DOWNLOAD["STORAGE_KWARGS"]["bucket"] = "{{ S3_GRADE_BUCKET }}"
 
 ORA2_FILEUPLOAD_BACKEND = "s3"
-FILE_UPLOAD_STORAGE_BUCKET_NAME = "{{ S3_STORAGE_BUCKET }}"
+FILE_UPLOAD_STORAGE_BUCKET_NAME = "{{ S3_FILE_UPLOAD_BUCKET }}"
```

Comment:
Just checking: is this just a refactor, or does it qualify as a fix? 

I am not 100% certain how `lms-env`, `cms-env`, and `openedx-common-settings` relate. If the _effective_ setting prior to this patch is that `S3_FILE_UPLOAD_BUCKET` is being honoured, then it's a refactor. If the original situation was that `S3_FILE_UPLOAD_BUCKET` was ignored and `S3_STORAGE_BUCKET` used instead, then it's a fix (and needs a changelog entry).

Reply:
I just double-checked the behavior before this patch. The settings in `lms-env` and `cms-env` were being ignored. So this is a **fix**. I will update the PR. Thanks for pointing it out. I thought it was the other way around :sweat_smile:

Reply:
I added an entry to the changelog. Let me know if it's ok.